manifests/alertmanager.pp

```diff
@@ -1,47 +1,45 @@
-# Class: prometheus::alertmanager
 #
-# This module manages prometheus alertmanager
+# @summary This module manages prometheus alertmanager
 #
-# Parameters:
-#  [*arch*]
+# @param arch
 #  Architecture (amd64 or i386)
```

Comment:
I think I've seen the need to correctly indent it with 3 spaces or it wasn't picked up. Not sure if that's also the case here.
```suggestion
#   Architecture (amd64 or i386)
```

Reply:
If I read the REFERENCE.md correctly the current indentation works fine.